modules/nf-core/deepcell/mesmer/main.nf

```diff
@@ -0,0 +1,40 @@
+process DEEPCELL_MESMER {
+    tag "$meta.id"
+    label 'process_medium'
+
+    // Setting up the container to latest all the time because why not.
+    container "vanvalenlab/deepcell-applications:0.4.0"
```

Comment:
no bioconda? 😢

Reply:
No, I don't think there is a conda for this at all. To be fair, since it's CNN based stuff with torch etc, I can see that bioconda recipe being a big pain to make...

Reply:
> I can see that bioconda recipe being a big pain to make.
You mean big FUN? 😛 If you make it, I'll approve it

Reply:
sharing is caring.